GiEnJul/Controllers/AdminController.cs

```diff
@@ -197,6 +206,7 @@ public async Task<ActionResult> SuggestConnectionAsync([FromBody] PostConnection
 
                 giver.IsSuggestedMatch = true;
                 giver.MatchedRecipient = connectionDto.RecipientRowKey;
+                giver.MatchedFamilyId = connectionDto.RecipientFamilyId;
```

Comment:
`giver.MatchedFamilyId = recipient.FamilyId` <- using the recipients familyid will keep the integrity of the familyid as it will always be correct. 

This solution will remove the need of passing the familyid from frontend, which means the connectionDto does not need any familyid related property